src/js/brandColors/brandColors.ts

```diff
@@ -0,0 +1,88 @@
+import { BrandColors } from './brandColors.types';
+
+export const brandColors: BrandColors = {
+  white: {
+    white000: '#ffffff',
```

Comment:
why are we doing `white000` and not `000`?

Reply:
As per our conversation and weighing up the pros and cons updated here https://github.com/MetaMask/design-tokens/pull/582/commits/5fab2d55d41430c23663f0debc4d9f886700210b